test/suites/container_devices_disk.sh

```diff
@@ -87,3 +89,43 @@ test_container_devices_raw_mount_options() {
   # shellcheck disable=SC2154
   deconfigure_loop_device "${loop_file_1}" "${loop_device_1}"
 }
+
+test_container_devices_disk_ceph() {
+  local LXD_BACKEND
+
+  LXD_BACKEND=$(storage_backend "$LXD_DIR")
+  if ! [ "${LXD_BACKEND}" = "ceph" ]; then
+    return
+  fi
+
+  RBD_POOL_NAME=lxdtest-$(basename "${LXD_DIR}")-disk
+  ceph osd pool create $RBD_POOL_NAME 1
```

Comment:
I'd consistently make this `${RBD_POOL_NAME}` rather than `$RBD_POOL_NAME` in the entire test script.

Reply:
And same for `$RBD_POOL_PATH`